app/scripts/modules/cluster/reports/forms/cluster.project.form.details.js

```diff
@@ -684,6 +694,28 @@ angular.module( 'ngm.widget.project.details', [ 'ngm.provider' ])
 					});
 				}, 
 
+				// open-close beneficiary target detail
+				openCloseDetailBeneficiaries:function(index){
+					if (!$scope.detailBeneficiaries[index]){
+						$('#openDB-' + index).hide();
+						$('#closeDB-' + index).show();
```

Comment:
if using ng-show on buttons, can we remove jquery hide/show?
```
$('#openDB-' + index).hide();
$('#closeDB-' + index).show();
```

Reply:
With new commit, i remove the rightnow remove the jquery hide and show